pandora-server-directory/src/account/accountDirectMessages.ts

```diff
@@ -1,11 +1,11 @@
 import { createHash } from 'crypto';
-import type { IClientDirectoryArgument, IClientDirectoryPromiseResult, IDirectoryClientArgument, IDirectoryDirectMessage, IDirectoryDirectMessageAccount, IDirectoryDirectMessageInfo } from 'pandora-common';
+import type { IClientDirectoryArgument, IClientDirectoryPromiseResult, IDirectoryClientArgument, IDirectoryDirectMessageAccount, IDirectoryDirectMessageInfo } from 'pandora-common';
 import { GetDatabase } from '../database/databaseProvider';
 import { Account, GetDirectMessageId } from './account';
 import { accountManager } from './accountManager';
 import { DatabaseDirectMessageInfo } from '../database/databaseStructure';
 
-const MESSAGE_LOAD_COUNT = 50;
+const MESSAGE_MAX_COUNT = 50;
```

Comment:
This should be moved to the `inputLimits.ts` file in pandora-common as it is no longer just an implementation detail, but greatly affects the behavior.